sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -40,7 +40,178 @@ az storage account create --name MyStorageAccount --resource-group MyResourceGro
+        {
+            // Specifying specific resources that failed, since its a directory transfer
+            // maybe only one file failed out of many
+            logStream.WriteLine($"Exception occured with TransferId: {args.TransferId}," +
+                $"Source Resource: {args.SourceResource.Path}, +" +
+                $"Destination Resource: {args.DestinationResource.Path}," +
+                $"Exception Message: {args.Exception.Message}");
+        }
+        return Task.CompletedTask;
+    };
+    TransferManager transferManager = new TransferManager(options);
+
+    // Create simple transfer directory upload job which uploads the directory and the contents of that directory
+    DataTransfer uploadDirectoryJobId = await transferManager.StartTransferAsync(
+        new LocalDirectoryStorageResourceContainer(sourcePath),
+        new BlobDirectoryStorageResourceContainer(container, Randomize("sample-blob-directory")));
+}
+```
+
+```C# Snippet:DownloadDirectory_EventHandler_ActiveDirectoryAuthAsync
+    // Create BlobTransferManager with event handler in Options bag
+    TransferManagerOptions options = new TransferManagerOptions();
+    ContainerTransferOptions downloadOptions = new ContainerTransferOptions();
+    downloadOptions.TransferFailed += async (TransferFailedEventArgs args) =>
+    {
+        // TODO: change the Exception if it's a RequestFailedException and then look at the exception.StatusCode
+        if (args.Exception.Message.Contains("500"))
+        {
+            Console.WriteLine("We're getting throttled stop trying and lets try later");
+        }
+        else if (args.Exception.Message == "403")
+        {
+            Console.WriteLine("We're getting auth errors. Might be the entire container, consider stopping");
+        }
+        // Remove stub
+        await Task.CompletedTask;
+    };
+    TransferManager transferManager = new TransferManager(options);
+
+    // Simple Download Directory Job where we upload the directory and it's contents
+    await transferManager.StartTransferAsync(
+        sourceDirectory, destinationDirectory);
+
+    // Create different download transfer
+    DataTransfer downloadDirectoryJobId2 = await transferManager.StartTransferAsync(
+        sourceDirectory2,
+        destinationDirectory2);
+}
+```
+
+```C# Snippet:CopySingle_ConnectionStringAsync
+    // Create simple transfer single blob upload job
+    DataTransfer transfer = await transferManager.StartTransferAsync(sourceResource, destinationResource);
+    await transfer.AwaitCompletion();
+
+    // Generous 10 second wait for our transfer to finish
+    CancellationTokenSource cancellationTokenSource = new CancellationTokenSource(10000);
+    await transfer.AwaitCompletion(cancellationTokenSource.Token);
+}
+```
+
+```C# Snippet:CopyDirectory
+    await transferManager.StartTransferAsync(
+        sourceDirectory1,
+        destinationDirectory1);
+}
+```
+
+```C# Snippet:ErrorHandlingPermissions
```

Comment:
Hm, is this from one of the samples I deleted? We might be able to remove this snippet since you already have a related one above.

It's also causing CI to fail
![image](https://user-images.githubusercontent.com/48961492/206570758-ece10a5b-18f5-4121-ad11-6d74ce6cd344.png)